### ISSUE TITLE
context and react-router + others

### ISSUE THREAD

**Issue by gmaclennan** [malte-wessel/minimal-flux#10]
I'm playing around with v0.5.0-alpha together with react-router v1.0.0-beta2, knowing full well that many things are going to break. I'm having trouble with context and `<FluxComponent>`. I also have [material-ui](www.material-ui.com) attaching itself to context.

Is v0.5.0-alpha only going to work with React 0.14 alpha right now?

**Comment by malte-wessel** [comment]
Yes, right there are some problems with the the owner/parent based context system. The alpha version is meant to be used with 0.14. Downside is that react-router only works with 0.13 :(
